cli/demo_direct_call.sh

```diff
@@ -89,6 +89,10 @@ echo "* Send ${AMOUNTTRANSFER} funds from Alice's incognito account to Bob's inc
 $CLIENT trusted --mrenclave ${MRENCLAVE} --direct transfer ${ICGACCOUNTALICE} ${ICGACCOUNTBOB} ${AMOUNTTRANSFER}
 echo ""
 
+echo "* Waiting 2 seconds"
+sleep 2
+echo ""
+
```

Comment:
As mentioned in the description, I re-added these sleeps in the demo/integration test scripts again, otherwise the calls would get executed simultaneously (= nonce clash), or a getter execution could take place before the call was executed.

Reply:
Could you please ~add a comment as well why the sleeps are necessary here?~
Or better yet extract it to a function named like `sleep_to_avoid_nonce_clash()`?

Reply:
Since only 1 out of 2 sleeps is made to avoid a nonce clash (the other is to prevent the getter from being executed too early), and we're in a shell script, would a comment still be acceptable?

Reply:
Absolutely, thanks :+1: